### ISSUE TITLE
ImportError: cannot import name 'sk_use_webauthn' from 'asyncssh.sk'

### ISSUE THREAD

**Issue by commonism** [ronf/asyncssh#719]
Hi,

running master, unit tests fail with importerror for sk_use_webauthn.

> python3 -m unittest tests/test_config.py 

```
E
======================================================================
ERROR: test_config (unittest.loader._FailedTest.test_config)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_config
Traceback (most recent call last):
  File "/usr/lib/python3.12/unittest/loader.py", line 137, in loadTestsFromName
    module = __import__(module_name)
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "~/workspace/asyncssh/tests/test_config.py", line 30, in <module>
    import asyncssh
  File "~/workspace/asyncssh/asyncssh/__init__.py", line 122, in <module>
    from . import sk_eddsa, sk_ecdsa, eddsa, ecdsa, rsa, dsa, kex_dh, kex_rsa
  File "~/workspace/asyncssh/asyncssh/sk_ecdsa.py", line 33, in <module>
    from .sk import sk_enroll, sk_sign, sk_webauthn_prefix, sk_use_webauthn
ImportError: cannot import name 'sk_use_webauthn' from 'asyncssh.sk' (~/workspace/asyncssh/asyncssh/sk.py)
```

running all of the unit tests may fix this as sk_use_webauthn is set at 

> ./tests/sk_stub.py:    asyncssh.sk.sk_use_webauthn = use_webauthn

**Comment by ronf** [comment]
I think I see the problem here. It looks like you'll get this error if you don't have the "fido" package installed. Adding the following should fix this:

```patch
diff --git a/asyncssh/sk.py b/asyncssh/sk.py
index 8dffdb0..ca5aef7 100644
--- a/asyncssh/sk.py
+++ b/asyncssh/sk.py
@@ -341,6 +341,7 @@ try:
                       not ctypes.windll.shell32.IsUserAnAdmin()
 except (ImportError, OSError, AttributeError): # pragma: no cover
     sk_available = False
+    sk_use_webauthn = False

     def _sk_not_available(*args: object, **kwargs: object) -> NoReturn:
         """Report that security key support is unavailable"""
```

**Comment by ronf** [comment]
This fix is now in the "develop" branch as commit 32a9fe1.

**Comment by ronf** [comment]
This fix is now available in AsyncSSH 2.19.0.
